lib/index.js

```diff
@@ -159,13 +159,14 @@ EslintValidationFilter.prototype.processString = function processString(content,
   };
 
   if (this.testGenerator) {
+    const result = filteredOutput.results[0];
     let messages = [];
 
     if (filteredOutput.results.length) {
-      messages = filteredOutput.results[0].messages || [];
+      messages = result.messages || [];
```

Comment:
I was unsure if it was guaranteed to be present. I will update to remove this defaulting.